Content.Server/Solar/EntitySystems/PowerSolarSystem.cs

```diff
@@ -59,9 +60,9 @@ internal sealed class PowerSolarSystem : EntitySystem
         public float TotalPanelPower = 0;
 
         /// <summary>
-        /// Queue of panels to update each cycle.
+        /// Timer used to avoid updating the panels every frame (which would be overkill)
         /// </summary>
-        private readonly Queue<SolarPanelComponent> _updateQueue = new();
+        private float _updateTimer = 0f;
```

Comment:
any reason this is changed? just so they all properly update at once?

Reply:
this is because I've seen the issue #10508 and thought of the queue as strange. Probably it was there for a reason.

Reply:
it was there for perf since theres tons of solar panels that dont need to update often but i'm fine not having the queue. it is a little strange

Reply:
Unless we get entity systems then system data should be killed.

Reply:
If you are referring to panel/sun rotation being in the system class I totally agree with you. In my opinion the sun should be an invisible entity (with a component specifying its rotation and velocity) and panels should be responsible for their rotation info. I can work on this, just let me know if in this pr or in a new one.

As for the queue:
The appearance's `SetData` is called for each solar panel. This is something that I was avoiding by using a single netevent. I can revert back to using the queue for offloading the coverage calculation to successive frames for performance (like in the original implementation). The decision is up to you.

Reply:
@mirrorcult I still need a feedback on this, queue or no queue?